src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractCollectionIntegrationTestBase.java

```diff
@@ -34,15 +34,19 @@ protected AbstractCollectionIntegrationTestBase(String collectionNamePrefix) {
   }
 
   @BeforeAll
-  public void createSimpleCollection() {
+  public void createDefaultCollection() {
```

Comment:
(not part of core changes)
Minor renaming to make it more clear this is the "default" collection that is normally created before IT.
May be overridden to change handling.